### DISCUSSION TITLE
error: end_date (set session.debug=True for tracebacks)

### DISCUSSION THREAD

**Discussion by raczo07** [rramosp/ai4eng.v1#165] [01 - INTRODUCCIÓN]
Hola cuando envio el ejemplo de suma de matrices LAB 01.01 - Practice submission me sale este error 
error: end_date (set session.debug=True for tracebacks)

**Comment by Felipe-RA** [comment]
Saludos raczo07

Si eres estudiante de la UdeA: 

La fecha de entrega para este laboratorio venció la fecha `2022-06-22 23:59`

Esto acorde a las fechas definidas y acordadas con los estudiantes, y publicadas en el calendario de entregas en la página del curso:

![image](https://user-images.githubusercontent.com/49454068/179059450-d0045ebe-21b4-48b5-9144-7a7c8fa64e62.png)

**Reply by raczo07** [reply]
Hola, no soy estudiante, estoy haciendo el curso de forma voluntaria

**Reply by raczo07** [reply]
Me registre en el formulario del curso UdeA de manera erronea, ya pude solucionar registrandome en el correcto.]

Muchas gracias

**Reply by Felipe-RA** [reply]
Me alegra que hayas solucionado tu problema!
